lib/browser/api/utility-process.ts

```diff
@@ -6,49 +6,46 @@ import { MessagePortMain } from '@electron/internal/browser/message-port-main';
 const { createProcessWrapper } = process._linkedBinding('electron_browser_utility_process');
 
 class IOReadable extends Readable {
-  _shouldPush: boolean = false;
-  _data: (Buffer | null)[] = [];
-  _resume: (() => void) | null = null;
-
-  _storeInternalData (chunk: Buffer | null, resume: (() => void) | null) {
-    this._resume = resume;
-    this._data.push(chunk);
-    this._pushInternalData();
+  #shouldPush: boolean = false;
+  #data: (Buffer | null)[] = [];
+  #resume: (() => void) | null = null;
+
+  storeInternalData (chunk: Buffer | null, resume: (() => void) | null) {
```

Comment:
i suppose this is not `#` because it needs to be called by C++? maybe keep the `_` just to indicate it's not intended to be called by a user.

Reply:
Yup, makes sense.